src/averageAges.js

```diff
@@ -15,11 +15,24 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const men = century !== undefined
+    ? people.filter(man => {
+      const isMan = man.sex === 'm';
+      const diedInThisCentury = century === Math.ceil(man.died / 100);
+
+      return isMan && diedInThisCentury;
+    })
+
+    : people.filter(man => {
+      const isMan = man.sex === 'm';
+
+      return isMan;
+    });
```

Comment:
You repeat yourself
You can make it easier
```suggestion
  const men = people.filter(man => {
      const isMan = man.sex === 'm';
      
      if (century) {
        const diedInThisCentury = century === Math.ceil(man.died / 100);
  
        return isMan && diedInThisCentury;
    }

      return isMan;
    });
```